lib/inc/drogon/CacheMap.h

```diff
@@ -268,6 +268,20 @@ class CacheMap
             fnOnInsert_(key);
     }
 
+    /**
+     * @brief Get the all values in the cache.
+     * */
+    std::vector<T2> getAllValues()
+    {
+        std::vector<T2> ret;
```

Comment:
A reserve here would be beneficial, after the lock to make sure the map size is locked in place as well